editor/filesystem_dock.cpp

```diff
@@ -69,9 +69,11 @@ bool FileSystemDock::_create_tree(TreeItem *p_parent, EditorFileSystemDirectory
 
 	// Create a tree item for the subdirectory.
 	TreeItem *subdirectory_item = tree->create_item(p_parent);
+	subdirectory_item->set_editable(0, true);
```

Comment:
I think this can be moved to `else` for the condition below. No reason to set something and then set back.